### PR TITLE
[TDA-6] Translate app to english

### PR DESCRIPTION
**Pull request by timesplash**
### Task TDA-6 Translate app to english

[Task link](https://andrey-tsvetkov.youtrack.cloud/issue/TDA-6/Translate-app-to-english)

### Short task description
Translate everything to english

### Readiness conditions
App is translated

- [ ] Unit-tests

### CODE REVIEW

**Review by aeDeaf — APPROVED** [review]

File: todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
- * Запрос на обработку регистации пользователя
+ * Register request.
+ * Allows for user to be registered
  */
 @Getter
 @NoArgsConstructor
 public class RegisterRequestDTO extends RequestDTO {
     /**
-     * имя
+     * name
```
Comment (Addm1X) [inline todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java:14]: One of reason is why i want to keep our development in english, is that verbose javadocs in Russian. Do you really need javadoc "name" to field name?)))) I don't think so))

File: todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java [hunk 2]
```diff
@@ -4,17 +4,18 @@
...
      */
     private String name;
     /**
-     * фамилия
+     * surname
```
Comment (Addm1X) [inline todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java:18]: same here, please consider this to all possible following verbose comments

File: todoapp-common/src/main/java/ru/todoapp/model/dto/RequestStatus.java [hunk 3]
```diff
@@ -1,15 +1,15 @@
 package ru.todoapp.model.dto;
 
 /**
- * Статус успешности выполнения запроса
+ * Request execution success status
```
Comment (Addm1X) [inline todoapp-common/src/main/java/ru/todoapp/model/dto/RequestStatus.java:4]: Now that is an appropriate javadoc.

File: todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java (hunk 1 above)
Comment (timesplash) [reply todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java:14]: Well tbh i wasnt in need of javadoc "Имя" for field name but @aeDeaf told me that fields in DTOs should have javadoc

Comment (Addm1X) [reply todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java:14]: Exactly, we can make your code self-documentive, without shitty useless javadocs

Comment (timesplash) [reply todoapp-common/src/main/java/ru/todoapp/model/dto/RegisterRequestDTO.java:14]: well, for now Im merging it, later we will fix it

**Pull request merged**